install/gcp/bootstrap/gcp_envoy_bootstrap.json

```diff
@@ -0,0 +1,142 @@
+{
+  "node": {
+    "id": "{{ .nodeID }}",
+    "cluster": "{{ .cluster }}",
+    "locality": {
+      {{ if .region }}
+      "region": "{{ .region }}",
+      {{ end }}
+      {{ if .zone }}
+      "zone": "{{ .zone }}",
+      {{ end }}
+      {{ if .sub_zone }}
+      "sub_zone": "{{ .sub_zone }}",
+      {{ end }}
+    },
+    "metadata": {{ .meta_json_str }}
+  },
+  "static_resources": {
+    "listeners": [
+      {
+        "name": "virtual",
+        "address": {
+          "socket_address": {
+            "address": "0.0.0.0",
+            "port_value": 15001
+          }
+        },
+        "use_original_dst": true,
+        "filter_chains": [
+          {
```

Comment:
Would be good to have some comments on what's different from the normal template.

I think we discussed in the past - having a separate file is safer, but keeping in sync may be trickier. If it's only the google grpc stuff - maybe we can add it in the main template as an option ?

Reply:
You are right, maybe just adding an option would be better. Let me try it and either come back with justification or another PR.

Reply:
I would rather merge this PR in 1.1, and have another PR in master. At least this one is very safe.